nixos/modules/services/hardware/monado.nix

```
@@ -0,0 +1,86 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  inherit (lib) mkDefault mkEnableOption mkIf mkOption mkPackageOption;
+
+  cfg = config.services.monado;
+
+in
+{
+  options.services.monado = {
+    enable = mkEnableOption "Monado wrapper and user service";
+
+    package = mkPackageOption pkgs "monado" { };
+
+    defaultRuntime = mkEnableOption "" // mkOption {
+      description = ''
+        When enabled, set Monado as the default OpenXR runtime on the system.
+      '';
+    };
+  };
+
+  config = mkIf cfg.enable {
+    security.wrappers."monado-service" = {
+      owner = "root";
+      group = "root";
```

Nit: I'd prefer explicit non-suid.

```suggestion
      setuid = false;
      owner = "root";
      group = "root";
```